dev/package-lock.json

```diff
@@ -16,8 +16,8 @@
             "integrity": "sha512-PDkSRY7KltW3M60hSBlerxI8SFPXsO3AL/aRVsO4Kh9IHRW74Ih75gUuTd/aE4LSSFqypb10UIX3QzOJwBQMGQ==",
             "dev": true,
             "requires": {
-                "@types/events": "*",
-                "@types/node": "*"
+                "@types/events": "1.2.0",
```

Comment:
Did you update this package lock before or after realizing you were on npm 5.6?

Reply:
Had thought it was after but obviously not.  Fixed.

Reply:
I think I did an npm update as well, hence still a lot of changes but not the same kind.